### PR TITLE
Fix HMR issue with webpack-stats-analyzer

### LINKED ISSUE #1143: issue with webpack-analyzer + HMR
I get the error below when developing with the webpack analyzer (configured as instructed in @timneutkens' example).

Code: https://github.com/relatenow/relate

```
$ nodemon -w server -w universal 
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /Users/sdubois/Documents/relate/server/**/* /Users/sdubois/Documents/relate/universal/**/*
[nodemon] starting `node ./server`
> Using external babel configuration
> location: "/Users/sdubois/Documents/relate/package.json"
> Using "webpack" config function defined in next.config.js.
c DONE  Compiled successfully in 15491ms

> Ready on http://localhost:3000
Webpack Bundle Analyzer is started at http://127.0.0.1:8888
Use Ctrl+C to close it
c WAIT  Compiling...

c DONE  Compiled successfully in 2941ms

Couldn't parse bundle asset "/Users/sdubois/Documents/relate/.next/4.e69f6923a6bd656a9415.hot-update.js".
Analyzer will use module sizes from stats file.

events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE 127.0.0.1:8888
    at Object.exports._errnoException (util.js:1022:11)
    at exports._exceptionWithHostPort (util.js:1045:20)
    at Server._listen2 (net.js:1262:14)
    at listen (net.js:1298:10)
    at doListening (net.js:1397:7)
    at _combinedTickCallback (internal/process/next_tick.js:77:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
[nodemon] app crashed - waiting for file changes before starting...
```

### PR DESCRIPTION
**Pull request by timneutkens**
Fixes #1143

### CODE REVIEW

File: examples/with-webpack-bundle-analyzer/next.config.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ module.exports = {
...
     config.plugins.push(
       new BundleAnalyzerPlugin({
+        analyzerMode: 'disabled',
         // For all options see https://github.com/th0r/webpack-bundle-analyzer#as-plugin
-        analyzerMode: 'server',
-        analyzerHost: '127.0.0.1',
-        analyzerPort: 8888,
-        openAnalyzer: false
+        generateStatsFile: true,
+        // Will be available at `./next/stats.json`
```
Comment (sedubois) [inline examples/with-webpack-bundle-analyzer/next.config.js]: `.next` not `./next`

File: examples/with-webpack-bundle-analyzer/README.md [hunk 2]
```diff
@@ -26,3 +26,5 @@ now
 ## The idea behind the example
 
 This example shows how to analyze the output bundles using [webpack-bundle-analyzer](https://github.com/th0r/webpack-bundle-analyzer#as-plugin)
+
+To view the stats use `webpack-bundle-analyzer .next/stats.json`
```
Comment (sedubois) [inline examples/with-webpack-bundle-analyzer/README.md]: You could add a script to package.json: `"viewBundle": "webpack-bundle-analyzer .next/stats.json"`

Comment (timneutkens) [inline examples/with-webpack-bundle-analyzer/README.md]: Good one 🙌

File: examples/with-webpack-bundle-analyzer/next.config.js (hunk 1 above)
Comment (timneutkens) [inline examples/with-webpack-bundle-analyzer/next.config.js]: Fixed 🙌

**Comment by nkzawa** [comment]
:+1:

**Pull request merged**